### PR TITLE
added whatsapp community link.

### PR DESCRIPTION
**Pull request by Aurazor**
Added whatsapp community link!
whatsapp

### CODE REVIEW

**Review by Saamiyah — APPROVED** [review]
LGTM!

File: src/components/HomeSocialPresence.astro [hunk 1]
```diff
@@ -38,14 +38,13 @@ import IconLinkedin from "./icons/icon-linkedin.astro"
...
-        href="https://www.facebook.com/groups/frontendcodersmu"
-        class="bg-[#4267B2] text-sm md:text-normal"
+        href="https://chat.whatsapp.com/CqXAj6m8NtQB6NN8B9aoEJ"
+        class="bg-[#25D366] text-sm md:text-normal"
     >
-        <p slot="tagline">Facebook Group</p>
-        <p slot="description">Our legacy presence on Facebook.</p>
-        
-        <IconFacebookGroup class="w-14 md:w-16 p-2" slot="icon"/>
+        <p slot="tagline">Whatsapp Group</p>
```
Comment (MrSunshyne) [inline src/components/HomeSocialPresence.astro:44]: Change the text from `Whatsapp Group` to `Whatsapp Community`

File: src/components/HomeSocialPresence.astro [hunk 2]
```diff
@@ -38,14 +38,13 @@ import IconLinkedin from "./icons/icon-linkedin.astro"
...
+        <p slot="description">Our Community on Whatsapp</p>
```
Comment (MrSunshyne) [inline src/components/HomeSocialPresence.astro:45]: Change the text to : 

`Join this community to stay informed about our events`

**Review by MrSunshyne — CHANGES_REQUESTED** [review]
Looks great! I left two comments about the text to fix. I should have included those in the initial issue.

Can you also add a description to the PR and a screenshot of the new look. It helps to understand what this PR did when we look at it after 1 year. 

Thanks!

**Comment by MrSunshyne** [comment]
Everything's looking good ! merging now. Thank you for the PR @Aurazor

**Review by MrSunshyne — APPROVED** [review]

**Pull request merged**